v2/oauth/multiple-frontends-with-separate-backends.mdx

```diff
@@ -0,0 +1,427 @@
+---
+title: Multiple frontend domains with separate backends
+hide_title: true
+---
+
+import BackendSDKTabs from "/src/components/tabs/BackendSDKTabs"
+import CoreInjector from "/src/components/coreInjector"
+
+# Multiple frontend domains with separate backends
+
+You can use the following guide if you have a single [**Authentication Service**](/docs/oauth/introduction#authentication-service) that is used by multiple applications.
+In turn, each app has separate **`frontend `** and **`backend`** instances that are served from different domains.
+
+Using the actual **OAuth 2.0** terminology, we can say that each **`frontend`** can be considered a [**Client**](/docs/oauth/introduction#client) and each **`backend`** is a [**Resource Server**](/docs/oauth/introduction#resource-server).
+
+:::info
+This guide assumes that you already have setup and configured **SuperTokens** in your [**Authentication Service**](/docs/oauth/introduction#authentication-service). 
+
+For more information on how to do that please check our [quickstart guide](/docs/thirdparty/introduction).
```

Comment:
Its not necessary that users would want to use third party login only. They may want to use email password or passwordless or third party or mfa etc.. So dont just link to third party.

Also, when the comment says "Authentication server" its a bit unclear what that means here. What i think is needed is a section in this page about architecture where we draw up a diagram with different parts - you will have 2 client applications (fe1, be1 and fe2, be2), one auth server which will comprise of fe and be with supertokens sdks and the core. Then we can say that we assume that you have already setup the authorization server with our sdks and the login methods that are needed for that, and can link to the various login methods, or to the guides page.

Reply:
will be resolved